src/controllers/accounts/AccountsConvertController.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+import { BadRequest } from 'http-errors';
+
+import { AccountsConvertService } from '../../services/accounts';
+import { IAddressParam, IConvertQueryParams } from '../../types/requests';
+import AbstractController from '../AbstractController';
+
+export default class AccountsConvertController extends AbstractController<AccountsConvertService> {
+	constructor(api: ApiPromise) {
+		super(api, '/accounts/:address/convert', new AccountsConvertService(api));
+		this.initRoutes();
+	}
+
+	protected initRoutes(): void {
+		this.safeMountAsyncGetHandlers([['', this.accountConvert]]);
+	}
+
+	private accountConvert: RequestHandler<
+		IAddressParam,
+		unknown,
+		unknown,
+		IConvertQueryParams
+	> = ({ params: { address }, query: { scheme, prefix, publicKey } }, res) => {
+		// Validation of the `scheme` query param
+		const cryptoScheme = !scheme ? 'sr25519' : scheme;
```

Comment:
Tiny nit, generally if possible I think it's better to have a truthy value be first followed by what the default value will be so its easier to read. Example:
`const cryptoScheme = scheme ? scheme : 'sr25519';`